iidm/iidm-api/src/main/java/com/powsybl/iidm/network/AreaType.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Marine Guibert {@literal <marine.guibert at artelys.com>}
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+public interface AreaType extends Identifiable<AreaType> {
+
+    @Override
+    default IdentifiableType getType() {
+        return IdentifiableType.AREA_TYPE;
+    }
+
+    boolean isMergeable(AreaType other);
```

Comment:
Same here, I don't think it is a good idea to have this method exposed in the interface.